### PR TITLE
PR - Improving DOCS + telegram plugin

### PR DESCRIPTION
**Pull request by raffus**
- New documentation examples about creating strategies.
- Fixing telegram plugin so mobile devices can call the methods using upper or lower case (now its case-insensitive). When using mobile phone devices, the first input was always upper case and the robot could not identify the requested command. Now this has been fixed.

### CODE REVIEW

File: docs/strategies/creating_a_strategy.md [hunk 1]
```diff
@@ -66,6 +66,70 @@ If you find out in the check function that you want to give a new advice to the
...
+
+    // For debugging purposes.
+    strat.log = function() {
+      // your code!
+    }
+
+    // Based on the newly calculated
+    // information, check if we should
+    // update or not.
+    strat.check = function() {
```
Comment (askmike) [inline docs/strategies/creating_a_strategy.md:113]: this function is passed the candle, if you don't specify it as an argument you cannot access `candle.close` on the next line. Could you add that?

**Comment by askmike** [comment]
Awesome, could you quickly fix the comment in the strategy example?

**Comment by raffus** [comment]
Done! :)

2017-07-31 15:24 GMT-03:00 Mike van Rossum <notifications@github.com>:

**Comment by askmike** [comment]
Great, thanks :)

**Comment by raffus** [comment]
Hey Mike, wouldn't it be good to merge the docs into stable?
That might reduce some issues requests.

**Comment by askmike** [comment]
It is already merged into stable, but as a rule all PRs (some hotfix
exceptions) need to to into develop first :)

**Pull request merged**